src/app/components/data-request/data-request.component.ts

```diff
@@ -1,79 +1,151 @@
 import { dataRequestCategory, dataRequestMenuItem } from "./data-request.menus";
-import { fields as requestFields } from "./data-request.schema.json";
-import { fields as annotationFields } from "./download-annotations.schema.json";
 
 @Component({
   selector: "baw-data-request",
   template: `
-    <h1>Data Request</h1>
-
-    <h2>Annotations Download</h2>
-
-    <p>To download a standard CSV of annotations</p>
-
-    <ol>
-      <li>Navigate to the project you're interested in</li>
-      <li>Then, choose the site you want to download annotations for</li>
-      <li>Finally, click the <i>Download annotations</i> link</li>
-    </ol>
-
-    <baw-wip>
-      <baw-form
-        title="Annotations Download"
-        subTitle="Please select the timezone for the CSV file containing annotations for ..."
-        submitLabel="Download Annotations"
-        [model]="annotationModel"
-        [fields]="annotationFields"
-        [submitLoading]="annotationLoading"
-        (onSubmit)="submitDownloadAnnotation($event)"
-      ></baw-form>
-      <baw-form
-        title="Custom Data Request"
-        subTitle="
-          Use this form to request a customized annotations list or other data related to the
-          audio recordings on this website. You <strong>do not need</strong> to use this form
-          if you need the standard <strong>annotations CSV</strong> download.
-        "
-        submitLabel="Submit"
-        [model]="requestModel"
-        [fields]="requestFields"
-        [submitLoading]="requestLoading"
-        (onSubmit)="submitDataRequest($event)"
-      ></baw-form>
-    </baw-wip>
+    <!-- Display content if there is no errors -->
+    <ng-container *ngIf="!error">
+      <h1>Data Request</h1>
+
+      <!-- Display forms if all loading completed -->
+      <ng-container *ngIf="!loadingModel && !error">
+        <h2 class="text-center">Annotations Download</h2>
+        <ng-container *ngIf="canRequestAnnotations; else placeholder">
+          <baw-request-annotations
+            [user]="user"
+            [site]="site"
+            [region]="region"
+          ></baw-request-annotations>
+        </ng-container>
+
+        <baw-request-custom></baw-request-custom>
+      </ng-container>
+    </ng-container>
+
+    <!-- Template placeholder for how to request annotations -->
+    <ng-template #placeholder>
+      <p>To download a standard CSV of annotations</p>
+      <ol>
+        <li>Navigate to the project you're interested in</li>
+        <li>Then, choose the site you want to download annotations for</li>
+        <li>Finally, click the <i>Download annotations</i> link</li>
+      </ol>
+    </ng-template>
+
+    <!-- Display error if exists -->
+    <ng-container *ngIf="error">
+      <baw-error-handler [error]="error"></baw-error-handler>
+    </ng-container>
+
+    <!-- Display spinner if loading -->
+    <ng-container *ngIf="loadingModel">
+      <baw-loading></baw-loading>
+    </ng-container>
```

Comment:
let's split this into two separate pages - there's no need for them to be on the same page